pytorch_lightning/trainer/trainer.py

```diff
@@ -91,7 +91,8 @@ def __init__(
             overfit_pct: float = 0.0,
             track_grad_norm: int = -1,
             check_val_every_n_epoch: int = 1,
-            fast_dev_run: bool = False,
+            fast_dev_run=None,  # backward compatible, todo: remove in v0.8.0
+            unit_test: bool = False,
```

Comment:
move this at the end as it kind of new and some users may use positional arguments